data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,58 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	(Stay for the next play.)`
+				`	(Leave.)`
+					goto leaveearly
+			`	Everyone takes their seats, and the Arach announcer heads to the stage again. "A cathartic tragedy, our third play of this evening is. With you: 'Til Death."`
+			`	The stage lights go dark as the curtains open to a single spotlight. On the right side of the stage it illuminates an old Arach getting dressed inside a small shack. He coughs, and a something akin to an organ begins to play a melancholic melody. Once he leaves the shack it's left in the dark, as the spotlight follows him to the middle of the stage where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight which goes dark as the perspective shifts and a second spotlight renews the focus, finally lighting up the left side of the stage where the Arach starts tending to a garden by the entrance of a manor. The piano stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead, plucking it and throwing it to the ground before leaving. The gardener picks up the flower, sticking it inside his clothes, and continues his work.`
+			`	Once his work is done, he heads back to his shack, the spotlights following him as he moves around the stage. Once inside his shack, he looks inside a small box, taking a simple vase, and placing the flower inside it. The stage goes dark as he goes to sleep, and the piano stops, only the occasional coughing fit echoing throughout the theater. When the light comes to the shack again, the Arach slowly gets up, and leaves for the gardening work again, pausing for a moment on the street part of the stage during a heavy coughing fit. Once he's done with the work again, and begins going back home, he starts coughing again, not letting up for more than a second at a time, during which the spotlights transition to the next section of the stage, and the piano plays a brief note. Once inside his shack, he collapses, knocking down the flower vase in the process. The piano plays a few slow notes, and the last spotlight goes out during the final note.`
+			`	You hear the curtains closing, and the lights of the theater turn back on again, as the audience is taken by a round of applause. Once they've settled, the announcer goes up to the stage again. "And with that, concluded, our soiree is. Thank you all for gracing us with your presence, I, in name of the theater, do." He lowers his head in a bow, and leaves the stage again, as nearly all of the patrons in the audience get up and head for the exit, this time with nobody else coming in to take their seats. At the same time, the orchestra starts packing up their instruments. There are, however, a few who remain in their seats, including the Arach who brought you here.`
+			choice
+				`	(Stay.)`
+					goto end
+				`	(Leave.)`
```

Comment:
Does this need a different branch than just `leaveearly`? In this case, there's not really many other Arachi patrons to watch you go.